Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM python:2.7
+FROM python:2.7-stretch
```

Comment:
why not move to python 3, if you're testing python 3 also in travis?

Reply:
Because I would like to test it more to be sure that nothing breaks. Test coverage here is not great and this has worked so far with python 2, so I would be more conservative.